README.md

```diff
@@ -100,141 +73,63 @@ Previously we suggested using prebuilt versions of the packages hosted on a `lbl
 Due to the inability of this system to appropriately support multiple versions, this service is deprecated and will be deactivated in the near future.
 Migration to the unpkg.com hosted version is easy and will allow you to control which version you want to use.
 
-The first step is to replace the script and stylesheet imports with the unpkg.com equivalents, e.g.:
-
-```diff
-- <script src="https://embeddable.gelinkt-notuleren.lblod.info/assets/frontend-embeddable-notule-editor.js"></script>
-+ <script src="https://unpkg.com/@lblod/embeddable-say-editor@~3/dist/assets/@lblod/embeddable-say-editor.js"></script>
-```
-
-You will also need to change your require statement to remove the outdated package name:
-
-```diff
- window.addEventListener('load', function() {
--  let App = require('frontend-embeddable-notule-editor/app').default.create({
-+  let App = require('@lblod/embeddable-say-editor/app').default.create({
-     autoboot: false,
-     name: 'embeddable-editor'
-   });
-```
+Simply refer to the basic example below for the basic setup. Instead of passing the plugin configuration to the "initEditor" function, 
+pass it directly to the `renderEditor` function and await it. The resulting object will be 
+fully initialized and ready for use.
 
 ## Basic Example: The editor in an HTML file 
 The idea is that you can have multiple HTML tags in which you can initialize an editor. We'll explain how it works and the process can be repeated if multiple editors are required. We need some HTML structure to start with, and then set-up the editor when everything has finished loading and rendering. We can also initialize the editor with some content or set it later via an event. Use something like the following HTML snippet as a base.  
-**Note**: the order of the JavaScript files matters.
 
 In this section, we will assume you are using unpkg, but using another service that serves npm modules or building and hosting the bundles yourself should work just as well. Simply link the corresponding files to the correct location for your setup.
 
-For an interactive example, refer to this [jsfiddle](https://jsfiddle.net/piemonkey/oLpv64c2/).
+For an interactive example, refer to this [jsfiddle](https://jsfiddle.net/abeforfiddle/7zugt5nv/25/).
```

Comment:
I think the `/25` is the step number from the original fork. I think you can set this version as the base, removing the need for this, so meaning we can update the fiddle in the future without having to update the url here.